content/articles/2018-12-pkgdown-1-2-0.Rmarkdown

```diff
@@ -0,0 +1,68 @@
+---
+title: 'pkgdown 1.2.0'
+author: Mara Averick
+date: '2018-12-04'
+slug: pkgdown-1-2-0
+description: > 
+  pkgdown 1.2.0 is now on CRAN!
+categories:
+  - package
+photo:
+  url: https://pixabay.com/en/decoration-packages-gifts-ribbons-3229259/
+  author: jackmac34
+---
+
+```{r setup, include = FALSE}
+library(pkgdown)
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+We're happy to announce that [pkgdown](https://pkgdown.r-lib.org/) 1.2.0 is now
+available on CRAN. pkgdown is designed to make it quick and easy to build a
+website for your package. Here, we'll highlight some of the new features, and
+improvements in this version. For a full list of changes, please see the
+[NEWS](https://github.com/r-lib/pkgdown/blob/master/NEWS.md#pkgdown-120).
+
+## New features
+
+The new
+[`deploy_site_github()`](https://pkgdown.r-lib.org/reference/deploy_site_github.html)
+function can be used to automatically deploy your package website to GitHub
+Pages with continuous integration systems (like
+[travis](https://travis-ci.org/)). Setup details can be found
+[here](https://pkgdown.r-lib.org/reference/deploy_site_github.html#setup).
```

Comment:
We are gradually moving all tidyverse sites to use this process so that they're always up-to-date.